binding/webgl_rendering_context.cc

```diff
@@ -986,6 +987,73 @@ napi_value WebGLRenderingContext::BufferData(napi_env env,
   return nullptr;
 }
 
+/* static */
+napi_value WebGLRenderingContext::BufferSubData(napi_env env,
+                                                napi_callback_info info) {
+  LOG_CALL("BufferSubData");
+  napi_status nstatus;
+
+  size_t argc = 3;
+  napi_value args[3];
+  napi_value js_this;
+  nstatus = napi_get_cb_info(env, info, &argc, args, &js_this, nullptr);
+  ENSURE_NAPI_OK_RETVAL(env, nstatus, nullptr);
+  ENSURE_ARGC_RETVAL(env, argc, 3, nullptr);
+
+  WebGLRenderingContext *context = nullptr;
+  nstatus = UnwrapContext(env, js_this, &context);
+  ENSURE_NAPI_OK_RETVAL(env, nstatus, nullptr);
+
+  ENSURE_VALUE_IS_NUMBER_RETVAL(env, args[0], nullptr);
```

Comment:
Nit - I generally try and do all the js-arg validation at once:

```c++
ENSURE_VALUE_IS_NUMBER_RETVAL(env, args[0], nullptr);
ENSURE_VALUE_IS_NUMBER_RETVAL(env, args[1], nullptr);
ENSURE_VALUE_IS_NUMBER_RETVAL(env, args[2], nullptr);
...
```

Reply:
Reformatted.  Ty for the pointer!